src/lib/bindings/http/validation/structured.js

```diff
@@ -32,17 +32,20 @@ function parse(payload, headers, receiver) {
     source: undefined
   };
 
+
+  const unprocessed = new Set(Object.keys(incoming));
```

Comment:
is `Set` necessary here.   Can't we assume that the keys in the object are unique already?

Reply:
I think they should be unique. The thing is: If I understand it correctly `Object.keys(incoming)` returns an array and deletion from an array is more complicated than just `delete()` which is offered by a `Set` object.